src/client/src/rt-theme/index.ts

```diff
@@ -1,15 +1,8 @@
-import type { Theme } from './themes'
+import { Theme } from './themes'
 import baseStyled, { ThemedStyledInterface } from 'styled-components'
-export type {
-  LightShade,
-  DarkShade,
-  ColorPalette,
-  AccentName,
-  AccentPaletteMap,
-  Color,
-} from './colors'
-export { themes, getThemeColor } from './themes'
-export type { Theme, TouchableIntentName, ColorProps, ThemeSelector } from './themes'
+export * from './colors'
+export * from './themes'
+export * from './themes'
```

Comment:
Same as above - is it more explicit to export specific types?

Reply:
Same here